### ISSUE TITLE
Create a tool to collect Google Accessibility information

### ISSUE THREAD

**Issue by joshi1983** [hhaccessibility/hhaccessibility.github.io#676]
We want a few Python scripts to help us gather accessibility information from Google. Google has some https://www.google.com/accessibility/blog/post/building-maps-everyone.html

Here are a few specific requirements:
- Written in Python(latest version preferred)
- All the new scripts should be stored in importers/google_places or importers/google_accessibility.  If it can be integrated with the existing scripts in importers/google_places, put them there.
- At least one of the scripts will write a .csv file.  You could have a separate script for downloading raw data to temporary files if that helps you reduce API quota during development and testing as many of our other tools in importers directory do.
- The csv file must contain at least the longitude, latitude, and name of each location.
- The csv file must also have at least 1 accessibility-related column.  Ideally, it has a column for every available accessibility question from Google.
- A yes/no or percentage for each accessibility question would be the best value for each question on each location.

Nice to haves:
- Include a column for Google Place ID with each location in the generated CSV.  This could be useful for filtering duplicate locations or merging accessibility information from this tool into information on the same location gathered by a separate import tool

We have several import tools for different data sources as you will find in the importers directory.  Each imports data from a different data source such as Google Places, OpenStreetMap, the yelp website.  Each generally works by interacting with their API's directly or by simulating a web browser and scraping information out of the site's HTML.  Interacting with an API is preferred but not always feasible.  This issue will involve researching or reverse engineering to find a way to do similar for downloading this accessibility information.

This issue is related to #357 which added an import tool found at importers/google_places which collected basic information on various Google Places but didn't capture any accessibility information.

**Comment by joshi1983** [comment]
@bhaswanthchava did you end up finding any accessibility information from Google Places even by manually navigating the site?  AccessLocator has a pitch soon and any information you found could help us go to the pitch with a better understanding of our competitors.
